requirements_doc.txt

```
@@ -2,19 +2,19 @@ importlib-metadata==4.13.0; python_version<'3.8'
 breathe>=4.34.0
 beautifulsoup4>=4.11.1
 lxml>=4.7.1
-myst-nb~=0.17.1
+myst-nb~=0.17.2
 
-Sphinx>=4.5.0,<5
+Sphinx>=4.5.0,<8
```

```suggestion
Sphinx>=4.5.0,<5
```
It is the documentation part that seems to fail. Revert these for now to separate out issues that make the PR fail